lang/ilias_it.lang

```diff
@@ -1945,7 +1945,6 @@ auth#:#login_page_switch_rte#:#Use Rich-Text-Editor
 auth#:#lti_consumer_inactive#:#Il consumatore dello strumento LTI è disabilitato.
 auth#:#saml_tab_head_idp#:#IDP
 awrn#:#awareness_now_online#:#Adesso online
-awrn#:#awareness_now_online_users#:#[ONLINE_USER_NAMES]
```

Comment:
Please only touch the `de` and `en` language files. This change has to be reverted.